lib/mule.js

```diff
@@ -113,11 +113,26 @@ function mulelink(guid) {
 	return l;
 }
 
-var VAULTORDER = [34, 32, 30, 28, 31, 33, 35,
-                  27, 25, 21, 20, 22, 26, 29,
-                  23, 17, 13, 11, 14, 18, 24,
-                  16,  9,  6,  4,  7, 10, 19,
-                  12,  5,  2,  1,  3,  8, 15];
+var VAULTORDER = [
```

Comment:
Ok. I haven't played in some time, so I have no idea what the vault situation looks like now...